web/src/main/java/org/bkvm/bookkeeper/BookkeeperManager.java

```diff
@@ -283,12 +283,11 @@ public void doRefreshMetadataCache() {
                 }
 
                 Iterable<Long> ledgersIds = bkAdmin.listLedgers();
+                metadataCache.clearLedgers(clusterId);
```

Comment:
I see 2 problems here:
1) if something goes wrong in the loop the database will be empty
2) if you have many ledgers for some time the users will see less data than expected (wrong results in the queries)

why do we have do it this way ?